command/event.go

```diff
@@ -24,33 +23,25 @@ Usage: consul event [options] [payload]
   a name, but a payload is optional. Events support filtering using
   regular expressions on node name, service, and tag definitions.
 
-Options:
-
-  -http-addr=127.0.0.1:8500  HTTP address of the Consul agent.
-  -datacenter=""             Datacenter to dispatch in. Defaults to that of agent.
-  -name=""                   Name of the event.
-  -node=""                   Regular expression to filter on node names
-  -service=""                Regular expression to filter on service instances
-  -tag=""                    Regular expression to filter on service tags. Must be used
-                             with -service.
-  -token=""                  ACL token to use during requests. Defaults to that
-                             of the agent.
-`
+` + c.Command.Help()
+
 	return strings.TrimSpace(helpText)
 }
 
 func (c *EventCommand) Run(args []string) int {
-	var datacenter, name, node, service, tag, token string
-	cmdFlags := flag.NewFlagSet("event", flag.ContinueOnError)
-	cmdFlags.Usage = func() { c.Ui.Output(c.Help()) }
-	cmdFlags.StringVar(&datacenter, "datacenter", "", "")
-	cmdFlags.StringVar(&name, "name", "", "")
-	cmdFlags.StringVar(&node, "node", "", "")
-	cmdFlags.StringVar(&service, "service", "", "")
-	cmdFlags.StringVar(&tag, "tag", "", "")
-	cmdFlags.StringVar(&token, "token", "", "")
-	httpAddr := HTTPAddrFlag(cmdFlags)
-	if err := cmdFlags.Parse(args); err != nil {
+	var name, node, service, tag string
+
+	f := c.Command.NewFlagSet(c)
+	f.StringVar(&name, "name", "",
+		"Name of the event.")
+	f.StringVar(&node, "node", "",
+		"Regular expression to filter on node names.")
+	f.StringVar(&service, "service", "",
+		"Regular expression to filter on service instances")
```

Comment:
Missing period at the end of this one. We keep an eye out for this and try to make them all match (include the period).